packages/react/src/ProgressBar/ProgressBar.features.stories.tsx

```diff
@@ -20,9 +20,9 @@ export const Color = () => <ProgressBar progress="66" bg="done.emphasis" aria-la
 
 export const MultipleItems = () => (
   <ProgressBar aria-valuenow={70} aria-label="Upload test.png">
-    <ProgressBar.Item progress={33} />
-    <ProgressBar.Item progress={23} sx={{backgroundColor: 'danger.emphasis'}} />
-    <ProgressBar.Item progress={14} sx={{backgroundColor: 'severe.emphasis'}} />
+    <ProgressBar.Item progress={33} bg={'accent.emphasis'} />
+    <ProgressBar.Item progress={23} bg={'danger.emphasis'} />
+    <ProgressBar.Item progress={14} bg={'severe.emphasis'} />
```

Comment:
Can we add a story with `sx` prop as well to keep track of backward compat?

Reply:
So with both bg and `sx`? Or do you think we should make `bg` optional?

Reply:
Just with sx so that we have a few examples of how it used to work.

> Or do you think we should make bg optional?

bg should definitely be optional, so that we don't break any existing instances

Reply:
I now just added sx to one of the three items, that should do it, right?